deploy/kubernetes/dolphinscheduler/values.yaml

```diff
@@ -583,6 +583,38 @@ worker:
   # -- Update strategy
   # ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
   #
+  # -- Direct Secret Mounting
+  # Mount secrets directly as environment variables
+  # Single secret
+  envFromSecret: ""
+  
+  # Multiple secrets (advanced use cases)
+  # Use when you have variables distributed across different secrets
+  # envFromSecrets:
+  #   - "git-sync-secret"        # Git synchronization credentials
+  #   - "database-secret"        # Database connection details
+  #   - "external-api-secret"    # Third-party API keys
+  
+  # -- Init Container for Advanced Processing
+  # Use when you need to transform, validate, or prepare configuration files
+  initContainers: {}
+    - name: config-processor
```

Comment:
This is a good example, but it should not be set as the default value for initcontainers. Instead, it should be treated as a comment.